README.md

```diff
@@ -25,9 +25,9 @@ include the following:
 
 Each release contains one zipped tarball per supported image above. The
 naming convention is
-`<base image with underscores><hyphen>ngx_http_datadog_module.so.tgz`,
-e.g. `nginx_1.23.1-alpine-ngx_http_datadog_module.so.tgz` or
-`amazonlinux_2.0.20230119.1-ngx_http_datadog_module.so.tgz`.
+`<base image with underscores>-<arch>-ngx_http_datadog_module.so.tgz`,
```

Comment:
Subjective, I found `<hyphen>` was not as straightforward as `-` and could be misleading. What's your thought?

Reply:
That's fine. `<hyphen>` I always had to read twice, like "what?"

My initial reasoning is that the "meta-syntax" `<thingy here>` uses angle brackets, and so using a hyphen literally might look like part of the meta-syntax instead of a literal hyphen.

Your way is better.